forte/processors/allennlp_processors.py

```diff
@@ -153,3 +165,39 @@ def _create_dependencies(input_pack, tokens, result):
                                   parent=tokens[heads[i] - 1],
                                   child=token)
             relation.rel_type = deps[i]
+
+    @staticmethod
+    def _create_srl(input_pack, tokens, result):
+        def parse_allennlp_srl_tags(tags):
+            pred_span = None
+            arguments = []
+            begin, end, prev_argument = None, None, ''
+            for i, item in enumerate(tags):
+                argument = '-'.join(item.split('-')[1:])
+                if prev_argument not in ('', argument):
+                    if prev_argument == 'V':
+                        pred_span = Span(tokens[begin].begin, tokens[end].end)
+                    else:
+                        arg_span = Span(tokens[begin].begin, tokens[end].end)
+                        arguments.append((arg_span, prev_argument))
+                prev_argument = argument
+                if item.startswith('B-'):
+                    begin = i
+                    end = i
+                if item.startswith('I-'):
+                    end = i
+            return pred_span, arguments
+        verbs = result['verbs']
+
+        for _, verb_item in enumerate(verbs):
+            pred_span, arguments = parse_allennlp_srl_tags(verb_item['tags'])
+            if arguments == []:
```

Comment:
There may be chances where we have the `Predicate`, but cannot find the arguments for it.  Note that this is still useful for downstream tasks.

Reply:
So i just remove line 194 and 195? But it won't be saved in PredicateLink. Is it OK?

Reply:
If there are no PredicateLink then yes, that's the expected output. We should not modify the output here.